cpython/search_pypi_top.py

```diff
@@ -38,6 +38,8 @@ def colored(msg, *ignored, **ignored2):
 
 
 IGNORE_CYTHON = True
+# Ignore these files from the initial directory listing
+IGNORED_FILENAMES = (".DS_Store",)
```

Comment:
Since it's a single filename, just rename it to "DS_STORE" and check "if filename != DS_STORE".

Maybe just add a short comment to explain what is this directory.

Reply:
Updated!